src/components/AppsMenu/components/ShortcutItem.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+
+import { useClient, useFetchShortcut } from 'cozy-client'
+import { splitFilename } from 'cozy-client/dist/models/file'
+import { makeStyles } from 'cozy-ui/transpiled/react/styles'
+import Buttons from 'cozy-ui/transpiled/react/Buttons'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import cx from 'classnames'
+
+import styles from 'styles/apps-menu.styl'
+
+const useStyles = makeStyles(() => {
+  return {
+    text: {
+      lineHeight: '22.5px',
+      fontSize: '12px',
+      fontWeight: 400
+    }
+  }
+})
+
+export const ShortcutItem = ({ shortcut }) => {
+  const client = useClient()
+  const classes = useStyles()
+  const { shortcutInfos, shortcutImg } = useFetchShortcut(client, shortcut._id)
+
+  const { filename } = splitFilename(shortcut)
+  const url = shortcutInfos?.data?.url || '#'
+
+  /**
+   * If we don't have iconMimeType, we consider that the icon is a binary svg.
+   * Otherwise we consider that the icon comes from Iconizer api so it is in base64 directly.
+   */
+  const icon = shortcut.attributes.metadata.icon
+  const iconMimeType = shortcut.attributes.metadata.iconMimeType
+
+  return (
+    <Buttons
+      height="auto"
+      component="a"
+      target="_blank"
+      variant="text"
+      href={url}
+      title={filename}
+      className={cx(styles['apps-menu-grid-item-wrapper'], 'u-bdrs-5')}
+      label={
+        <div className={styles['apps-menu-grid-item']}>
+          <div className={styles['apps-menu-grid-item-icon']}>
+            {icon ? (
+              <img
+                src={
+                  iconMimeType
+                    ? `data:${iconMimeType};base64,${icon}`
+                    : `data:image/svg+xml;base64,${window.btoa(icon)}`
```

Comment:
```suggestion
                    : `data:image/svg+xml;base64,${btoa(icon)}`
```

can also be extracted in a helper like getIconSrc